librz/util/syscmd.c

```diff
@@ -118,7 +118,7 @@ static char *showfile(char *res, const int nth, const char *fpath, const char *n
 	return res;
 }
 
-RZ_API RZ_OWN char *rz_syscmd_ls(RZ_NONNULL const char *input) {
+RZ_API RZ_OWN char *rz_syscmd_ls(RZ_NONNULL const int argc, const char **argv) {
```

Comment:
I recommend just moving this function to the `cmd_shell.c`, there is no need to keep it in `librz/util/`

Reply:
Then you can make it `RZ_IPI` as well

Reply:
Are you sure? Because from what I can see, in the code within librz/core/cmd/cmd_shell.c, called RZ_API functions are all located in librz/util/ such as: rz_name_validate_char, rz_str_appendlen, rz_sys_get_environ, rz_sys_info, rz_str_array_join, rz_file_copy, including rz_syscmd_ls

Reply:
Yes, definitely sure, move it.